Kitodo/src/main/java/org/kitodo/production/forms/createprocess/FileUploadDialog.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.production.forms.createprocess;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.charset.Charset;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.LinkedList;
+import java.util.List;
+
+import javax.xml.parsers.ParserConfigurationException;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.kitodo.api.MdSec;
+import org.kitodo.api.Metadata;
+import org.kitodo.api.schemaconverter.DataRecord;
+import org.kitodo.api.schemaconverter.FileFormat;
+import org.kitodo.api.schemaconverter.MetadataFormat;
+import org.kitodo.config.OPACConfig;
+import org.kitodo.exceptions.ConfigException;
+import org.kitodo.exceptions.ParameterNotFoundException;
+import org.kitodo.exceptions.ProcessGenerationException;
+import org.kitodo.exceptions.UnsupportedFormatException;
+import org.kitodo.production.helper.Helper;
+import org.kitodo.production.helper.TempProcess;
+import org.kitodo.production.services.ServiceManager;
+import org.kitodo.production.services.data.ImportService;
+import org.primefaces.event.FileUploadEvent;
+import org.primefaces.model.UploadedFile;
+import org.w3c.dom.Document;
+import org.xml.sax.SAXException;
+
+public class FileUploadDialog extends MetadataImportDialog {
+
+    private UploadedFile file;
```

Comment:
I think since you handle the uploaded file directly in the `handleFileUpload` listener, this variable here is never used and therefor can be removed.
```suggestion
```